source/compatibility.tex

```diff
@@ -60,7 +60,7 @@
 Programs which depend upon \tcode{sizeof('x')} are probably rare.
 
 Subclause \ref{lex.string}:
-\change String literals made const\\
+\change String literals made const.\\
```

Comment:
My patch does not change the "const" \tcode style, it just adds periods etc. (At least I hope so.) I believe we have a separate issue for const vs. \tcode.